modin/pandas/utils.py

```diff
@@ -37,7 +37,7 @@ def from_non_pandas(df, index, columns, dtype):
     modin.pandas.DataFrame
         Converted DataFrame.
     """
-    from modin.data_management.factories.dispatcher import FactoryDispatcher
+    from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
```

Comment:
move?

Reply:
Even if we can, this should go in a separate issue/PR